lib/util/fs/index.ts

```diff
@@ -151,6 +174,22 @@ export async function readLocalDirectory(path: string): Promise<string[]> {
   return fileList;
 }
 
+export function readLocalDirectorySync(path: string): string[] | null {
```

Comment:
no sync, use async io

Reply:
This is used by `extract` and `extract` itself is not async though. what would you suggest in this case?

Reply:
make extract async